tests/lib/rules/no-legacy-test-waiters.js

```diff
@@ -0,0 +1,88 @@
+const rule = require('../../../lib/rules/no-legacy-test-waiters');
+
+const { ERROR_MESSAGE } = rule;
+const RuleTester = require('eslint').RuleTester;
+
+const ruleTester = new RuleTester({
+  parserOptions: {
+    ecmaVersion: 6,
+    sourceType: 'module',
+  },
+});
+
+ruleTester.run('no-legacy-test-waiters', rule, {
+  valid: [
```

Comment:
We need a test case that checks that the legacy test wait functions are allowed if they are not from the right import source.

Reply:
👍